src/icons/bear.svg

```diff
@@ -1,789 +1,117 @@
-</g>
-<g id="Head">
-	<circle cx="482" cy="449.9" r="200"/>
-	<rect x="341.9" y="431" width="122" height="355.3"/>
-	<rect x="541.8" y="431" width="140.2" height="355.3"/>
-	<rect x="456.2" y="431" class="st6" width="140.2" height="355.3"/>
-	<circle class="st6" cx="378.6" cy="421.1" r="270.4"/>
-	<path d="M367.8,453.9L626.6,700c0,0,43.2-108.4,39.6-315.9L416.4,276.4L367.8,453.9z"/>
-	<polygon class="st6" points="244.3,484.6 244.3,372.4 244.3,336.6 245.8,250.9 399.5,250.9 465.6,317 579,317 579,415.3 579,439.6 
-		564.7,428.1 553.2,441 537.5,429.6 526,442.4 514.6,431 460.3,485.4 	"/>
-	<path class="st7" d="M465.6,317c0,0-21.1,98.3,113.3,98.3V317H465.6z"/>
-	<polygon points="553.2,316.6 579,316.6 579,345.2 	"/>
-	<polyline class="st8" points="579,387.1 559.4,402.4 456,402.4 456,371.9 	"/>
-	<ellipse cx="559.1" cy="346.1" rx="3" ry="3"/>
-	<path d="M552.4,336.4c0,1.7-1.3,3-3,3c-1.7,0-3-1.3-3-3c0-1.7,1.3-3,3-3C551.1,333.4,552.4,334.7,552.4,336.4z"/>
-	<ellipse cx="540.3" cy="345.6" rx="3" ry="3"/>
-	<path d="M421.3,333.8c0-6.7,5.4-12.2,12.2-12.2c6.7,0,12.2,5.4,12.2,12.2"/>
-	<path class="st8" d="M357.1,290.5c0-23.4-19-42.4-42.4-42.4c-23.4,0-42.4,19-42.4,42.4"/>
-	<path d="M338.6,290.5c0-13.2-10.7-24-24-24c-13.2,0-24,10.7-24,24"/>
-	<rect x="107.8" y="431" class="st6" width="140.2" height="355.3"/>
-	<circle cx="679.5" cy="430.3" r="17.8"/>
-	<g>
-		<g>
-			<polyline class="st8" points="475.6,521.9 466.2,538.3 456.7,521.9 			"/>
-			<polyline class="st8" points="493.8,521.9 484.4,538.3 474.9,521.9 			"/>
-			<polyline class="st8" points="512,521.9 502.6,538.3 493.1,521.9 			"/>
-			<polyline class="st8" points="530.2,521.9 520.7,538.3 511.3,521.9 			"/>
-			<polyline class="st8" points="548.4,521.9 538.9,538.3 529.5,521.9 			"/>
-			<polyline class="st8" points="566.5,521.9 557.1,538.3 547.7,521.9 			"/>
-			<polyline class="st8" points="584.7,521.9 575.3,538.3 565.9,521.9 			"/>
-			<polyline class="st8" points="330.1,521.9 320.7,538.3 311.3,521.9 			"/>
-			<polyline class="st8" points="348.3,521.9 338.9,538.3 329.4,521.9 			"/>
-			<polyline class="st8" points="366.5,521.9 357.1,538.3 347.6,521.9 			"/>
-			<polyline class="st8" points="384.7,521.9 375.3,538.3 365.8,521.9 			"/>
-			<polyline class="st8" points="402.9,521.9 393.4,538.3 384,521.9 			"/>
-			<polyline class="st8" points="421.1,521.9 411.6,538.3 402.2,521.9 			"/>
-			<polyline class="st8" points="439.2,521.9 429.8,538.3 420.4,521.9 			"/>
-			<polyline class="st8" points="457.4,521.9 448,538.3 438.6,521.9 			"/>
-		</g>
-	</g>
-	<g>
-		<g>
-			<polyline class="st9" points="474.4,538.3 465,554.6 455.5,538.3 			"/>
-			<polyline class="st9" points="492.6,538.3 483.2,554.6 473.7,538.3 			"/>
-			<polyline class="st9" points="510.8,538.3 501.4,554.6 491.9,538.3 			"/>
-			<polyline class="st9" points="529,538.3 519.5,554.6 510.1,538.3 			"/>
-			<polyline class="st9" points="547.2,538.3 537.7,554.6 528.3,538.3 			"/>
-			<polyline 
```

Comment:
Embedded styles like this are supposed to work, but IE being IE sometimes has problems rendering these things. Even modern versions like IE11. I still haven't figure out the exact scenario, but I would recommend just testing out rendering of your icons in IE and if you see any problems with fill color or stroke, you might need to move these styles as attributes or inline styles down below.